src/Areas/LoadTesting/Services/LoadTestingService.cs

```diff
@@ -0,0 +1,287 @@
+            throw new Exception($"Failed to retrieve Azure Load Test Run: {loadTestRunResponse}");
+        }
+
+        var testRuns = new List<TestRun>();
+        await foreach (var binaryData in loadTestRunResponse)
+        {
+            var testRun = JsonSerializer.Deserialize<TestRun>(binaryData.ToString(), LoadTestJsonContext.Default.TestRun);
+            if (testRun != null)
+            {
+                testRuns.Add(testRun);
+            }
+        }
+
+        if (testRuns.Count == 0)
+        {
+            throw new Exception($"No test runs found for test ID '{testId}' in Load Test '{testResourceName}'.");
+        }
+        return testRuns;
+    }
+
+    public async Task<TestRun> CreateOrUpdateLoadTestRunAsync(string subscriptionId, string testResourceName, string testId, string? testRunId = null, string? oldTestRunId = null, string? resourceGroup = null, string? tenant = null, string? displayName = null, string? description = null, bool? debugMode = false, RetryPolicyOptions? retryPolicy = null)
+    {
+        ValidateRequiredParameters(subscriptionId, testResourceName, testRunId);
+        var loadTestResource = await GetLoadTestResourcesAsync(subscriptionId, resourceGroup, testResourceName, tenant, retryPolicy);
+        if (loadTestResource == null)
+        {
+            throw new Exception($"Load Test '{testResourceName}' not found in subscription '{subscriptionId}' and resource group '{resourceGroup}'.");
+        }
+        var dataPlaneUri = loadTestResource[0]?.DataPlaneUri;
+        if (string.IsNullOrEmpty(dataPlaneUri))
+        {
+            throw new Exception($"Data Plane URI for Load Test '{testResourceName}' is not available.");
+        }
+
+        var credential = await GetCredential(tenant);
+        var loadTestClient = new LoadTestRunClient(new Uri($"https://{dataPlaneUri}"), credential);
+
+        TestRunRequest requestBody = new TestRunRequest
+        {
+            TestId = testId,
+            DisplayName = displayName ?? "TestRun_" + DateTime.UtcNow.ToString("dd/MM/yyyy") + "_" + DateTime.UtcNow.ToString("HH:mm:ss"),
+            Description = description,
+            DebugLogsEnabled = debugMode ?? false,
+            RequestDataLevel = debugMode == true ? RequestDataLevel.ERRORS : RequestDataLevel.NONE,
+        };
+
+        var loadTestRunResponse = await loadTestClient.BeginTestRunAsync(0, testRunId, RequestContent.Create(JsonSerializer.Serialize(requestBody, LoadTestJsonContext.Default.TestRunRequest)), oldTestRunId: oldTestRunId);
+        if (loadTestRunResponse == null)
+        {
+            throw new Exception($"Failed to retrieve Azure Load Test Run: {loadTestRunResponse}");
+        }
+
+        var loadTestRun = loadTestRunResponse.WaitForCompletionAsync().Result.Value.ToString();
+        return JsonSerializer.Deserialize<TestRun>(loadTestRun, LoadTestJsonContext.Default.TestRun) ?? new TestRun();
+    }
+    /* Load Test Run APIs End */
+
+    /* Load Test APIs Start */
```

Comment:
I don't think these comments are needed.  Also check formatting for all files.  Make sure there are empty lines between methods.